nextcord/application_command.py

```diff
@@ -2007,11 +2028,21 @@ def __init__(
         ] = description_localizations
         self.guild_ids_to_rollout: Set[int] = set(guild_ids) if guild_ids else set()
         self.use_default_guild_ids: bool = guild_ids is MISSING and not force_global
-        self.dm_permission: Optional[bool] = dm_permission
         self.default_member_permissions: Optional[
             Union[Permissions, int]
         ] = default_member_permissions
         self.nsfw: bool = nsfw
+        self.integration_types = (
+            None if integration_types is None else [IntegrationType(x) for x in integration_types]
+        )
+
+        if dm_permission is not None:
```

Comment:
DeprecationWarning?

Reply:
I was thinking this argument could be removed as it's a breaking change and the next version would be 3.0 anyway.

Reply:
I've removed the `dm_permission` parameter. The PR should be ready to merge.